views/event.ejs

```diff
@@ -647,7 +822,92 @@
             </div>
         </div>
     </div>
+
+    <div class="modal propose-session-modal" id="propose-session-modal" role="dialog" aria-hidden="true">
+
+        <div class="modal-dialog">
+            
+            <div class="modal-content">
+                
+                <div class="modal-header">
+                    <h4 class="modal-title">Propose a Session</h4>
+                </div>
+
+                <div class="modal-body">
+                    <div class="row modal-row">
+                        
+                        <div class="input-topic-box col-lg-12 no-padding">
+
+                            <div class="col-lg-12 no-padding">
+                                <div class="form-group">
+                                    <label class="col-lg-3 control-label no-padding session-title-label" for="topic_title">Session Title</label>
+
+                                    <div class="col-lg-9 no-padding">
+                                        <input class="form-control input-topic-title" type="text" name="topic_title" placeholder="Enter session title in less than 80 characters"value="" id="topic_title">
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="col-lg-12 no-padding">
+                                 <hr/> 
+                            </div>
+                        </div>
+
+                        <div class="session-preview col-lg-12 col-md-12 col-sm-12 col-xs-12">
+
+                        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 proposed-session-box">
```

Comment:
Corresponding changes are addressed here https://github.com/drewww/unhangout/commit/507ed1982ad5b1fcb0a4d7e90b9837ff874893b3 I think, by now I have a clear understanding of where we don't require duplicate classes :)